backend/lib/controllers/trustee-appointments/trustee-appointments.controller.test.ts

```diff
@@ -31,19 +32,19 @@ describe('TrusteeAppointmentsController', () => {
     }
 
     mockUseCase = {
-      getTrusteeAppointments: jest.fn(),
-    } as unknown as jest.Mocked<TrusteeAppointmentsUseCase>;
+      getTrusteeAppointments: vi.fn(),
+    } as unknown as Mocked<TrusteeAppointmentsUseCase>;
 
     (
-      TrusteeAppointmentsUseCase as jest.MockedClass<typeof TrusteeAppointmentsUseCase>
+      TrusteeAppointmentsUseCase as unknown as MockedClass<typeof TrusteeAppointmentsUseCase>
```

Comment:
It seems odd that we have to cast a class to unknown to cast it as a mocked version of itself.